lib/FinderNavigation/styles.scss

```diff
@@ -54,6 +54,16 @@ $finder-indent-level-support: 10 !default;
   width: 100%;
 }
 
+.finder__item--drag-wrapper {
```

Comment:
I'm not too sure this name encapsulates what this is really doing. Unless I am reading it wrong. `finder__item` is the entire row including sub children (looking at the story).

So does this add a drag wrapper style around the whole row and sub rows? Or is it just for the individual row?

------

Maybe if it was added to the story it'd be easier to understand...which then brings to the next piece of feedback;

Is this standalone class used within being bound to a prop? If so it'll be difficult to communicate how we use it in a components namespace I think. For example;

```
<FinderItem.Content isDraggable /> // has a class of .finder__item--draggable within the component
```

I believe this communicates the intent much easier and the story can reflect that, thoughts?